packages/orbit-components/src/Textarea/index.tsx

```diff
@@ -57,69 +59,75 @@ const Textarea = React.forwardRef<HTMLTextAreaElement, Props>((props, ref) => {
-          error ? "shadow-form-element-error" : "shadow-form-element",
-          disabled ? "cursor-not-allowed" : "cursor-text",
-          fullHeight && "h-full flex-1",
-          disabled || readOnly
-            ? "bg-form-element-disabled-background"
-            : [
-                "bg-form-element-background",
-                error ? "hover:shadow-form-element-error-hover" : "hover:shadow-form-element-hover",
-              ],
-          "[&::placeholder]:text-form-element-foreground",
+      <label ref={labelRef}>
+        {label && (
+          <FormLabel
+            error={!!error}
+            help={!!help}
+            required={required}
+            iconRef={iconRef}
+            onMouseEnter={() => setTooltipShownHover(true)}
+            onMouseLeave={() => setTooltipShownHover(false)}
+          >
+            {label}
+          </FormLabel>
         )}
-        data-state={getFieldDataState(!!error)}
-        data-test={dataTest}
-        aria-required={!!required}
-        id={inputId}
-        name={name}
-        value={value}
-        defaultValue={defaultValue}
-        disabled={disabled}
-        placeholder={String(placeholder)}
-        maxLength={maxLength}
-        onChange={onChange}
-        rows={rows}
-        onFocus={handleFocus}
-        onBlur={onBlur}
-        tabIndex={tabIndex ? Number(tabIndex) : undefined}
-        readOnly={readOnly}
-        ref={ref}
-        aria-describedby={shown ? `${inputId}-feedback` : undefined}
-        aria-invalid={error ? true : undefined}
-        {...dataAttrs}
-      />
+
+        <textarea
+          className={cx(
+            "w-full appearance-none",
+            "font-base text-normal p-300 leading-normal",
+            "rounded-100 relative box-border block overflow-auto",
+            "min-h-form-box-normal",
+            "duration-fast transition-shadow ease-in-out",
+            "border border-transparent",
+            // TODO: remove when will be migrated from tmp-folder
+            resize === RESIZE_OPTIONS.VERTICAL ? "resize-y" : "resize-none",
+            error ? "shadow-form-element-error" : "shadow-form-element",
+            disabled ? "cursor-not-allowed" : "cursor-text",
+            fullHeight && "h-full flex-1",
+            disabled || readOnly
+              ? "bg-form-element-disabled-background"
+              : [
+                  "bg-form-element-background",
+                  error
+                    ? "hover:shadow-form-element-error-hover"
+                    : "hover:shadow-form-element-hover",
+                ],
+            "[&::placeholder]:text-form-element-foreground",
+          )}
+          data-state={getFieldDataState(!!error)}
+          data-test={dataTest}
+          aria-required={!!required}
+          id={inputId}
+          name={name}
+          value={value}
+          defaultValue={defaultValue}
+          disabled={disabled}
+          placeholder={String(placeholder)}
```

Comment:
When going through the component using SR, without tooltip placeholder content is read, with tooltip is not read. that's little bit weird IMO. 🤔  not telling it's wrong/correct, just point for discussion (?).

![Snímek obrazovky 2025-03-05 v 16 28 45](https://github.com/user-attachments/assets/daa93900-5d67-4cdb-a05d-c4de43fb3e74)
![Snímek obrazovky 2025-03-05 v 16 28 30](https://github.com/user-attachments/assets/e63eda98-bf2b-4f0f-a40c-858068299e69)

Reply:
I tried to "fix" this but it always reads it the same. I've checked different textareas and it seems that once there is `aria-describedby` set, it doesn't read the placeholder 🤔 So honestly not sure what else to do here, do you have any suggestion?